database/factories/TrackFactory.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+use App\User;
+use App\Reciter;
+use App\Album;
+use App\Track;
+use Faker\Generator as Faker;
+
+$factory->define(Track::class, function (Faker $faker) {
+    return [
+        'name' => $name = $faker->name,
+        'slug' => str_slug($name),
+        'reciter_id' => function () {
+            return factory(Reciter::class)->create();
+        },
+        'album_id' => function () {
+            return factory(Album::class)->create();
```

Comment:
return `->id()` here, like in `created_by`. Make sure all other instances of this are correct too.